src/App.css

```diff
@@ -39,52 +68,199 @@
 
 .bookContainer {
   display: flex;
+  flex-direction: row;
   justify-content: space-between;
-  padding: 4rem;
-  border: 1px solid black;
-  gap: 1rem;
-  align-items: center;
+  background-color: #fff;
+  margin: 1% 5%;
+  padding: 1% 10% 1% 2%;
+  font-family: 'Roboto Slab', serif;
+  border: solid 1px #e8e8e8;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
 }
 
-.buttonsList {
+.bookTitle {
   display: flex;
+  flex-direction: column;
   gap: 1rem;
 }
 
-.bookTitle {
+.category {
+  color: #121212;
+  opacity: 0.5;
+  font-weight: bold;
+}
+
+.title {
+  color: #121212;
+  font-size: larger;
+  font-weight: bold;
+}
+
+.author {
+  color: #4386bf;
+}
+
+.buttonsList {
   display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-  width: 20rem;
+  gap: 1rem;
 }
 
 .buttonsList li {
   list-style: none;
 }
 
-.bookChart {
+.bookBtn {
+  outline: none;
+  border: none;
+  background-color: #fff;
+  color: #4386bf;
+  cursor: pointer;
+  padding-left: 0%;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+}
+
+.progress {
   display: flex;
-  gap: 0.5rem;
-  width: 10rem;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  min-width: 35%;
 }
 
-.bookChapter {
+.chart {
   display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-  width: 20rem;
+  flex-direction: row;
+  align-items: center;
```

Comment:
- [ ] Please 🙏🏽 add the styles assigned in the original design such as font family, margin, padding, and colors to be similar to the Zeplin design.